automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -149,6 +149,51 @@ class RawlsApiSpec
     Rawls.parseResponse(Rawls.getRequest(s"${Rawls.url}api/submissions/queueStatus"))
 
   "Rawls" - {
+    "should agree with Sam on the permissions it sets on its resources" in {
+      implicit val token: AuthToken = owner.makeAuthToken()
+      val resourceTypes = Sam.config.listResourceTypes()
+
+      resourceTypes.collect {
+        case ResourceType(typeName, roles, _, _, _, _) if typeName.equals("workspace") =>
+          roles.collect {
+            case ResourceRole(roleName, actions, descendantRoles, _) if roleName.equals("owner") =>
+              actions should contain allElementsOf(List("share_policy::reader", "share_policy::share-reader", "share_policy::writer", "share_policy::share-writer", "share_policy::can-compute", "share_policy::owner", "write", "own", "compute", "delete", "read_auth_domain", "update_auth_domain"))
+              descendantRoles.get("google-project") shouldBe Option(Set("owner"))
+            case ResourceRole(roleName, actions, descendantRoles, _) if roleName.equals("reader") =>
+              actions should contain allElementsOf(List("read", "read_policy::owner", "read_auth_domain"))
+              actions should not contain allElementsOf(List("write", "compute"))
```

Comment:
anyElementsOf? That is, neither "write" nor "compute" should be in the list of actions.

Reply:
Yeah exactly

Reply:
But is there a checker for that? I would expect that "should not contain allElementsOf" tests that BOTH "write" and "compute" aren't in the list… not that neither of them is.

Reply:
Ohh I understand, your expectation was correct. Fixed it by switching to `noElementsOf`